### PR TITLE
Update PRS examples

### PR DESCRIPTION
**Pull request by alainli0928**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

**Review by cloga — APPROVED** [review]

File: cli/jobs/pipelines/iris-batch-prediction-using-parallel/pipeline.yml [hunk 1]
```diff
@@ -10,36 +10,32 @@ tags:
...
-  job_out_file:
-    type: uri_file
-    mode: rw_mount
-
 jobs:
   batch_prediction:
     type: parallel
     compute: azureml:cpu-cluster
     inputs:
-      input_data: ${{parent.inputs.job_data_path}}
```
Comment (wangchao1230) [inline cli/jobs/pipelines/iris-batch-prediction-using-parallel/pipeline.yml:33]: why do we remove the binding?

File: sdk/jobs/pipelines/1g_pipeline_with_parallel_nodes/pipeline_with_parallel_nodes.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+#---------- 1. Connect to Azure Machine Learning Workspace ----------
```
Comment (bupt-wenxiaole) [inline sdk/jobs/pipelines/1g_pipeline_with_parallel_nodes/pipeline_with_parallel_nodes.py:1]: Can we cover this file in CI/CD?

**Comment by bupt-wenxiaole** [comment]
please fix the notebook checker failure.

File: cli/jobs/pipelines/iris-batch-prediction-using-parallel/pipeline.yml (hunk 1 above)
Comment (alainli0928) [reply cli/jobs/pipelines/iris-batch-prediction-using-parallel/pipeline.yml:33]: Want to reduce some complexity when user start to learn PRS job in pipeline.

**Review by bupt-wenxiaole — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**